docs/how-to/create-custom-image-using-existing-docker-image.md

```diff
@@ -0,0 +1,202 @@
+## How to create customized Docker images using an existing docker image.
```

Comment:
Change title to: Creating customized Docker images using an existing Docker image